compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4028,17 +4028,23 @@ OMR::X86::TreeEvaluator::ibyteswapEvaluator(TR::Node *node, TR::CodeGenerator *c
    return target;
    }
 
-enum BinaryArithmeticOps : uint32_t
+enum ArithmeticOps : uint32_t
    {
-   BinaryArithmeticInvalid,
+   ArithmeticInvalid,
    BinaryArithmeticAdd,
    BinaryArithmeticSub,
    BinaryArithmeticMul,
    BinaryArithmeticDiv,
    BinaryArithmeticAnd,
    BinaryArithmeticOr,
    BinaryArithmeticXor,
-   NumBinaryArithmeticOps
+   NumBinaryArithmeticOps,
+   UnaryArithmeticMin,
+   UnaryArithmeticMax,
```

Comment:
I'm confused.  Aren't Min and Max binary ops?   They call the binary vector evaluator but seem to be classified as unary ops.

Reply:
[Fixed](https://github.com/eclipse/omr/pull/6599/commits/96677434e35a7c2dc06d41d0d69aa94e24a27784)